x-pack/plugins/apm/server/routes/storage_explorer/is_cross_cluster_search.test.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Setup } from '../../lib/helpers/setup_request';
+import { isCrossClusterSearch } from './is_cross_cluster_search';
+import { ApmIndicesConfig } from '@kbn/observability-plugin/common/typings';
+
+describe('isCrossClusterSearch', () => {
+  it('returns false when there are no remote clusters in APM indices', () => {
+    const mockedSetup = {
+      indices: {
+        transaction: 'traces-apm*',
+        span: 'traces-apm*',
+        metric: 'metrics-apm*',
+        error: 'logs-apm*',
+      } as ApmIndicesConfig,
+    } as unknown as Setup;
+
+    expect(isCrossClusterSearch(mockedSetup)).toBe(false);
```

Comment:
```suggestion
    expect(isCrossClusterSearch(mockedSetup)).toBeFalsy()
```

Reply:
This will still pass with all falsy values (`false`, `0`, `''`, `null`, `undefined`, and `NaN`)